### PR TITLE
removing commenting settings

### PR DESCRIPTION
**Pull request by sewagodimo**
Removing all the changes I made to add commenting settings and reverted the migrations.

### CODE REVIEW

File: molo/commenting/tests/test_models.py [hunk 1]
```diff
@@ -126,74 +125,3 @@ def test_auto_remove_for_non_remove_flag(self):
...
-        self.french = SiteLanguageRelation.objects.create(
-            language_setting=Languages.for_site(self.site),
-            locale='fr',
-            is_active=True)
-        self.translated_article = self.mk_article_translation(
-            self.article1, self.french,
-            title=self.article1.title + ' in french',
-            subtitle=self.article1.subtitle + ' in french')
-
-    def test_get_comments_anonymous(self):
```
Comment (KaitCrawford) [inline molo/commenting/tests/test_models.py]: I think you can leave this test (and whatever setup is required for it) here. It's good to have

**Review by KaitCrawford — COMMENTED** [review]
@sewagodimo I think there's one of the tests that you can leave in, which should help the test coverage. I'm also not sure this migration is necessary. What happens if you take it out?

**Comment by codiebeulaine** [comment]
@KaitCrawford agreed. @sewagodimo 👍 once this passes :) 🍰

**Review by KaitCrawford — APPROVED** [review]
:+1: :cake:

**Pull request closed without merging**